src/components/dashboard/Claim.js

```diff
@@ -416,7 +416,9 @@ const getStylesFromProps = ({ theme }) => {
   }
 
   const learnMoreLink = {
-    cursor: 'pointer',
+    ...Platform.select({
+      web: { cursor: 'pointer' },
```

Comment:
is spread operator won't crash for native ? i suppose it will return undefined in that case so i'm not sure spread will work correctly, have you checked this ?

Reply:
all changes have been tested while developing them, it doesn't crash when spreading. tested on both iOS and android